cli/src/test/cli.spec.ts

```diff
@@ -28,7 +28,7 @@ test('cli with no args and cli help and cli --help give same output', async (t)
 // TODO(arik): Do we want to enforce a particular order?
 test('cli with no args lists all commands', async (t) => {
   const expectedCommands = new Set([
-    'deploy', 'destroy', 'download', 'help', 'list', 'logs', 'rename',
+    'deploy', 'destroy', 'download', 'help', 'list',  'lock', 'logs', 'rename', 'unlock',
```

Comment:
Shouldn't `lock` and `unlock` be hidden commands?

Reply:
I still see them.  Perhaps you're missing a push to this PR?